Classes/UT3HellbenderRearGunPawn.uc

```diff
@@ -39,10 +40,96 @@
 
 class UT3HellbenderRearGunPawn extends ONSPRVRearGunPawn;
 
+Var rotator SpineDrive;
+Var rotator ThighDriveL,ThighDriveR;
+Var rotator CalfDriveL,CalfDriveR;
+Var rotator FootDriveL,FootDriveR;
+Var rotator ArmDriveL,ArmDriveR;
+Var rotator ForeArmDriveL, ForeArmDriveR;
+Var rotator NeckDrive;
```

Comment:
Same comment here: either make these local variables inside the `AttachDriver` function, or...

Reply:
C:\Unreal Anthology\UT2004\UT3Vehicles\Classes\UT3HellbenderRearGunPawn.uc(48) : Error, 'Var' is not allowed here

Reply:
Make them `local` if you want to go that way.